web/ui/dashboard/src/app/private/pages/project/apps/apps.component.html

```diff
@@ -31,13 +43,15 @@
 					<button class="button__clear" (click)="showAppStatusDropdown = false; showOverlay = false">Clear</button>
 				</div>
 			</div>
-		</div>
+		</div> -->
 	</div>
 	<button class="button button__primary button__small" id="create-app-modal-button" (click)="showCreateAppModal = !showCreateAppModal">Create An App</button>
 </div>
 
 <div class="card">
-	<div class="table" id="apps-table-container">
+	<convoy-table-loader id="events_loader_loader" [tableHead]="appsTableHead" *ngIf="isloadingApps"></convoy-table-loader>
+
+	<div class="table" id="apps-table-container" *ngIf="!isloadingApps && displayedApps && displayedApps.length > 0">
 		<div class="table--container">
 			<table id="table">
```

Comment:
<img width="1132" alt="Screenshot 2022-06-02 at 4 31 20 PM" src="https://user-images.githubusercontent.com/17768868/171665674-5579ba43-8308-4a70-bdf0-3fffcc8cc752.png">

Please remove the date grouping on this app's table, it doesn't really work here.